examples/java/src/main/java/org/apache/beam/examples/snippets/transforms/io/gcp/bigquery/BigQueryMyData.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.examples.snippets.transforms.io.gcp.bigquery;
+
+// [START bigquery_my_data]
+
+import com.google.api.services.bigquery.model.TableRow;
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.apache.beam.sdk.coders.AvroCoder;
+import org.apache.beam.sdk.coders.DefaultCoder;
+
+class BigQueryMyData {
+
+  static class MyStruct {
+    String stringValue;
```

Comment:
You can add `@SuppressWarnings("unused")`. I also know that annotations and I think being `public` would do it. It is possible that Javadoc might cause it to be "used".

Reply:
I tried the `@SuppressWarnings("unused")` annotation on both the fields and the `MyStruct` class, as well as making everything `public` and adding Javadocs, but I'm still getting the error from `:examples:java:spotbugsMain`.

```
> ./gradlew :examples:java:spotbugsMain
Configuration on demand is an incubating feature.

> Task :examples:java:compileJava
Note: /Users/dcavazos/src/beam/examples/java/src/main/java/org/apache/beam/examples/snippets/transforms/io/gcp/bigquery/BigQueryMyData.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
[main] INFO org.gradle.internal.nativeintegration.services.NativeServices - Initialized native services in: /Users/dcavazos/.gradle/native

> Task :examples:java:spotbugsMain
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

> Task :examples:java:spotbugsMain FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':examples:java:spotbugsMain'.
> SpotBugs rule violations were found. See the report at: file:///Users/dcavazos/src/beam/examples/java/build/reports/spotbugs/main.html
```

Reply:
Ah, yes. It is (slightly) different for spotbugs. For that you would use `@SuppressFBWarnings("URF_UNREAD_FIELD")` (just guessing at the bug code - you can see which one in the report)

Reply:
Thank you! That finally fixed it